### PR TITLE
Update GitHub Issue Templates

### LINKED ISSUE #705: Update GitHub Issue and PR templates
##### ISSUE TYPE

 - Other

##### SUMMARY
Implement a new issue and PR template model to
Push users to refer to the documentation, closed issues, project discussions, and slack channel
Force them add pertinent supporting info to all tickets

### PR DESCRIPTION
**Pull request by ryanmerolle**
Closes #705 

@rodvand @sc68cal @nahun @jeremystretch 

What are your thoughts on: 

https://github.com/ryanmerolle/ansible_modules/issues/new/choose
https://github.com/ryanmerolle/ansible_modules/blob/devel/.github/pull_request_template.md

I pulled from the core NetBox project and NetBox-Docker

### CODE REVIEW

**Comment by rodvand** [comment]
I have been meaning to do something with this, so I'm all for a bit more structure in the issues/pull requests! Very good!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    about: "Please refer to the documentation before raising a bug or feature request."
+  - name: 📖 Contributing Policy
+    url: https://github.com/netbox-community/ansible_modules/blob/devel/CONTRIBUTING.md
+    about: "Please read through our contributing policy before opening an issue or pull request"
+  - name: ❓ Discussion
+    url: https://github.com/netbox-community/ansible_modules/discussions
+    about: "If you're just looking for help, try starting a discussion instead"
+  - name: 💬 Community Slack
+    url: https://netdev.chat/
+    about: "Join #netbox on the NetDev Community Slack for assistance with installation issues and other problems"
```
Comment (rodvand) [inline .github/ISSUE_TEMPLATE/config.yml:15]: Use #ansible here as well?

**Comment by ryanmerolle** [comment]
Can I please get an approval to merge?

**Comment by nahun** [comment]
Nice, thanks for that work!

**Review by rodvand — APPROVED** [review]

**Pull request merged**